pynobo.py

```diff
@@ -155,6 +156,92 @@ class API:
         DICT_NAME_TO_OVERRIDE_MODE = {NAME_NORMAL : OVERRIDE_MODE_NORMAL, NAME_COMFORT : OVERRIDE_MODE_COMFORT, NAME_ECO : OVERRIDE_MODE_ECO, NAME_AWAY : OVERRIDE_MODE_AWAY}
         DICT_NAME_TO_WEEK_PROFILE_STATUS = {NAME_ECO : WEEK_PROFILE_STATE_ECO, NAME_COMFORT : WEEK_PROFILE_STATE_COMFORT, NAME_AWAY : WEEK_PROFILE_STATE_AWAY, NAME_OFF : WEEK_PROFILE_STATE_OFF}
 
+    class Model:
+        """
+        A device model that supports Nobø Ecohub.
+
+        Lists of devices:
+        https://help.nobo.no/en/user-manual/before-you-start/what-is-a-receiver/list-of-receivers/
+        https://help.nobo.no/en/user-manual/before-you-start/what-is-a-transmitter/list-of-transmitters/
+        """
+
+        THERMOSTAT = "thermostat"
+        SWITCH = "switch"
+        CONTROL_PANEL = "control_panel"
+        UNKNOWN = "unkown"
+
+        def __init__(
+                self,
+                model_id: str,
+                type: Union[THERMOSTAT, SWITCH, CONTROL_PANEL, UNKNOWN],
+                name: str,
+                set_comfort: bool=True,
+                set_eco: bool=True,
+                has_temp_sensor: bool=False
+        ):
+            self._model_id = model_id
+            self._type = type
+            self._name = name
+            self._set_comfort = set_comfort
+            self._set_eco = set_eco
+            self._has_temp_sensor = has_temp_sensor
+
+        @property
+        def model_id(self) -> str:
+            """Model id of the component (first 3 digits of the serial number)."""
+            return self._model_id
+
+        @property
+        def name(self) -> str:
+            """Model name."""
+            return self._name
+
+        @property
+        def type(self) -> Union[THERMOSTAT, SWITCH, CONTROL_PANEL, UNKNOWN]:
+            """Model type."""
+            return self._type
+
+        @property
+        def set_comfort(self) -> bool:
```

Comment:
Names are hard... I really feel like writing xxx.set_comfort(24) and yyy.set_eco(18). Do you think a name like `has_comfort`, `supports_comfort`, `can_set_comfort` or something along those lines might make sense?

Reply:
I agree it is a bad name. I like your `supports_comfort` proposal.